### DISCUSSION TITLE
WebGIS Workshop 2024 in Graz

### DISCUSSION THREAD

**Discussion by gis-eni** [e-netze/webgis-community#162] [Announcements]
### Agenda:

Bitte ein Rating.
Für die einzelnen Punkte bitte ein ``Reply`` schreiben und so kommentieren, was genau besprochen/gezeigt werden sollte.

Tag 1: Vorträge von Energie Steiermark
Tag 2: Uservorträge (Liste bitte erweitern)

**Comment by gis-eni** [comment]
### Tag 1: WebGIS 6.0

Neue Funktionen in WebGIS 6.0
Migration von WebGIS 5.0 nach WebGIS 6.0

**Reply by faulgurke** [reply]
Die neuen Funktionen und auch der Umstieg erscheinen uns wichtige Inhalte für den Workshop zu sein.
Details: Wie funktioniert der Umstieg? Wie sehen die unterschiedlichen Skins aus? Wie werden WebGIS5 und WebGIS6 dokumentiert (Unterschiede, Gemeinsamkeiten)?

**Reply by fanie-mak** [reply]
Für wäre auch noch wichtig: Was ändert sich für den Anwender?

**Reply by marko-jer** [reply]
Wie schaut es mit den Suchen im "Kartenportal" und bei "Dienste hinzufügen", du hast bei einem WebGIS-Wednesday erwähnt, das es da Überlegungen gibt, die Suchen zu verbessern.

**Reply by Andreas-Lampert** [reply]
Wie schauts bei WebGIS 6 mit der Projektion 3857 aus? Gibt es Änderungen zu WebGIS 5?

**Comment by gis-eni** [comment]
### Tag 1: OpenSource

Wo geht die Reise hin?

**Reply by faulgurke** [reply]
Eine kurze Information über "wohin geht die Reise" ist zweckmäßig und wichtig
Wie soll die weitere Entwicklung aussehen?
Wer könnte bei der Weiterentwicklung mitarbeiten?
Will man die Weiterverbreitung von WebGIS ankurbeln?
Was ändert sich genau an den vertraglichen Grundlagen?

**Reply by Andreas-Lampert** [reply]
Wie geht es mit dem Support Vertrag weiter? Gibt es da weitere Neuigkeiten?

**Comment by gis-eni** [comment]
### Tag 1: Linux/Kubernetes

**Reply by faulgurke** [reply]
Für uns ein sehr wichtiges Thema.
Details wie das aussehen könnte, würden uns sehr interessieren.
Ist der container rootless?
Gibt es Prototypen, die man ausprobieren könnte?
Braucht es ein externes filesystem?

**Reply by fanie-mak** [reply]
Für uns interessant, vor allem Docker-Technologie (angedacht)

**Reply by faulgurke** [reply]
Und dann noch eine Frage: Müsste bei einer Containerisierung von WebGIS nicht auch gView containerisiert werden?

**Comment by gis-eni** [comment]
### Tag 1: WebGIS Hilfe

Hilfe gemeinsam verbessern. Wie geht das?

**Reply by faulgurke** [reply]
Um welche WebGIS Hilfe geht es konkret? Tutorial oder Manual?
Es ist sicher zweckmäßig den Ablauf einer Änderung/Verbesserung  einmal vorzuzeigen.

**Reply by fanie-mak** [reply]
Zweckmäßig beschreibt es am besten

**Comment by gis-eni** [comment]
### Tag 1: DataLinq

**Reply by faulgurke** [reply]
Für uns (mich) ein wichtiges Thema.
Es wäre schön würden ein paar Beispiele zur Verfügung gestellt.
Interessant wäre auch zu erfahren, welche Anwendungen die Kolleginnen und Kollegen bereits realisiert haben.

**Reply by fanie-mak** [reply]
Wir würden uns wünschen, dass die Zusammenarbeit noch verstärkt wird. z.B.: Code Beispiele auf das datalinq-forum auf github

**Reply by Andreas-Lampert** [reply]
Weitere Code Beispiele im DataLinq-Forum für die unterschiedlichen Werkzeuge bzw. in Kombination mit z.B. DataLinqHelper wären super.

**Comment by gis-eni** [comment]
### Tag 1: Javascript / D3

**Reply by faulgurke** [reply]
Steht im Zusammenhang mit dem obigen Punkt. Es gibt zahllose Möglichkeiten wie man JS/D3 mit DataLinq verbinden kann. Die Vorstellung von Beispielen wäre super.

**Reply by Andreas-Lampert** [reply]
Vorteile / Nachteile Verwendung chart.js vs. d3.js

**Comment by gis-eni** [comment]
### Tag 2: DataLinq - Marko, David

**Reply by faulgurke** [reply]
Wir freuen uns schon darauf!

**Reply by marko-jer** [reply]
Der Klimapass - ein Blick hinter die Kulissen

**Reply by fanie-mak** [reply]
weitere DataLinq-Beispiele Kärnten 

- Klima-Report (CCCA - https://klimaszenarien.at/oeks-15/)
- Bevölkerungsprognosen

**Comment by gis-eni** [comment]
### Tag 2: GIP / Rechenraum - Klaus

**Reply by faulgurke** [reply]
Klaus wird die Gehzeitberechnung herzeigen.

**Comment by gis-eni** [comment]
### Tag 2: gView 6.0 - Jürgen G

Neuerungen/Änderungen

**Reply by faulgurke** [reply]
gView hat sich in VBG zur wichtigen und relevanten Stütze bezüglich WebGIS entwickelt.
Ja bitte!

**Comment by fanie-mak** [comment]
Tag 2: Diskussion/Ideensammlung: STAC Erweiterung für WebGIS
Kurzvorstellung STAC Kärnten
Rasterdownload über WebGIS?

**Reply by gis-eni** [reply]
Abkürzungen bitte beschreiben. Was ist STAC? ;)

**Reply by fanie-mak** [reply]
STAC - Spatio Temporal Asset Catalog --> es werden Daten räumlich und zeitlich über einen Katalog im Internet angeboten. ;) 
Bei uns in Kärnten, ist dieser als API zugänglich, Landingpage https://gis.ktn.gv.at/api/stac/v1/
